moveit_core/online_signal_smoothing/src/butterworth_filter.cpp

```diff
@@ -100,8 +100,11 @@ bool ButterworthFilterPlugin::initialize(rclcpp::Node::SharedPtr node, moveit::c
 {
   node_ = node;
   num_joints_ = num_joints;
-  auto param_listener = std::make_unique<online_signal_smoothing::ParamListener>(node_);
-  auto filter_coeff = param_listener->get_params().butterworth_filter_coeff;
+  double filter_coeff = 1.5;
```

Comment:
@AndyZe Why do we need to initialize here ?
Wouldn't the param be available anyway since it is using 'generate_parameter_library` now

Reply:
Good point, I'll delete this.